src/entities/tokens.ts

```diff
@@ -114,3 +114,15 @@ export interface RainbowToken extends Asset {
 export interface IndexToken extends Asset {
   amount: string;
 }
+
+export interface TokenMetadata {
+  address: EthereumAddress;
+  name: string;
+  symbol: string;
+  color?: string;
+  isRainbowCurated?: boolean;
+  isVerified?: boolean;
+  shadowColor?: string;
+  chainId?: number;
+  decimals?: number;
```

Comment:
If you delete these why are they here?

Reply:
ts stuff: https://cloud.skylarbarrera.com/Screen-Shot-2022-11-14-12-44-13.38.png

if there's a better way im all 👂